uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -39,8 +39,12 @@ <h4 class="modal-title">New Document</h4>
       </div>
     </div>
     <div class="container">
-      <uib-tabset>
```

Comment:
Why are you getting rid of uib-tabset? I don't think you mentioned it in the technical details